manim/mobject/types/vectorized_mobject.py

```diff
@@ -898,10 +900,10 @@ def get_nth_curve_function(self, n: int) -> typing.Callable[[float], np.ndarray]
         """
         return bezier(self.get_nth_curve_points(n))
 
-    def get_nth_curve_function_with_length(
+    def get_nth_curve_length(
```

Comment:
If this function is necessary, it should probably just be
```py
def get_nth_curve_length(self, n, **kwargs):
    _, length = self.get_nth_curve_function_with_length(n, **kwargs)

    return length
```
And then `get_arc_length` should probably be updated to use it

Reply:
Done. I don't think `get_arc_length` needs to be updated since making it use get_nth_curve_length would make it less efficient than the current implementation, since it would also need to use `get_num_curves` as well.